### ISSUE TITLE
Add support for Kubernetes v1.16

### ISSUE THREAD

**Issue by LQBing** [kubernetes/dashboard#4327]
##### Environment

```
Installation method: kubectl 
Kubernetes version: v1.16.0
Dashboard version: kubernetesui/dashboard:v2.0.0-beta4
Operating system: ubuntu 18.04.1
Node.js version ('node --version' output): n/a
Go version ('go version' output): n/a
```

##### Steps to reproduce

Choose Workloads -> Deployments -> a deployment, click 'edit'. 
![image](https://user-images.githubusercontent.com/6857508/65488550-fc15c500-dedb-11e9-9af7-1c9ac3b4f883.png)

##### Observed result

The deployment yaml content not load, and I find there is a 404 request in the browser dev panel.
##### Expected result

It need display deployment resource yaml content.

**Comment by anshulahuja98** [comment]
Can you perhaps also share the console output as well?

**Comment by RnkeZ** [comment]
Hi, I have the same problem here is the console output.
![image](https://user-images.githubusercontent.com/1742984/65542443-7ad82580-df0f-11e9-949a-cb727e1f2d4b.png)

**Comment by anshulahuja98** [comment]
I replicated your dev environment. 
The issue seems to be with using Kubernetes version: v1.16.0
That's a fairly recent release, not sure if we should worry about that version as of now. 
The current dashboard release v2.0.0-beta4 only claims to support till 1.15 k8s.
And this feature works fine on 1.15

/awaiting-more-evidence

**Comment by RnkeZ** [comment]
Tkanks for the info.
Do we have ETA on v1.16.0 support?

**Comment by anshulahuja98** [comment]
I cannot comment on that
@maciaszczykm @floreks can perhaps give better info on this.

**Comment by floreks** [comment]
It all depends on client-go updating their code to support v1.16.

**Comment by zerkms** [comment]
Client go **DOES** support v1.16

**Comment by floreks** [comment]
@zerkms I don't see any official release that would support `v1.16` on their github page: https://github.com/kubernetes/client-go#compatibility-matrix

**Comment by LQBing** [comment]
![image](https://user-images.githubusercontent.com/6857508/65571613-e9f95c80-df97-11e9-814a-745df0983be2.png)

![image](https://user-images.githubusercontent.com/6857508/65573234-e2888200-df9c-11e9-9f5a-4b90a2991edd.png)

@anshulahuja98 here is the console output and kubernetes-dashboard pod log.

**Comment by floreks** [comment]
We've established already that it is related to some changes in K8S `v1.16`. We have to wait for the official release of `client-go` to support it. Right now we do not support it.

**Comment by discostur** [comment]
I think this is because of the API deprecation:

https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

**Comment by liggitt** [comment]
> We've established already that it is related to some changes in K8S `v1.16`. We have to wait for the official release of `client-go` to support it. Right now we do not.

Just to note, the replacement APIs (apps/v1) have been available since v1.9, and any client-go version since then can be used to access them. You don't need to wait for the v1.16 version of client-go, though that is available as well at https://github.com/kubernetes/client-go/tree/kubernetes-1.16.0
